frontend/integration-tests/tests/olm/crd.scenario.ts

```diff
@@ -0,0 +1,36 @@
+import { browser, $, by, ExpectedConditions as until, element } from 'protractor';
+import { appHost, checkLogs, checkErrors } from '../../protractor.conf';
+import * as crdView from '../../views/crd.view';
+import * as olmCatalogView from '../../views/olm-catalog.view';
+
+describe('Test for CRD related pages', () => {
+
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  it('View instances list of crd/catalogsource', async() => {
+    browser.get(`${appHost}/k8s/cluster/customresourcedefinitions`);
+    await browser.wait(until.presenceOf(crdView.crdItemsList));
+    await crdView.clickOneCrd('CatalogSource', 'catalogsources.operators.coreos.com');
+    await crdView.clickActionItem('View Instances');
+    await browser.wait(until.presenceOf(olmCatalogView.entryRows));
+    expect(olmCatalogView.entryRows.count()).toBeGreaterThan(0);
+  });
+
+  it('Check catalogsource instance without OperatorGroup has correct message', async() => {
+    browser.get(`${appHost}/k8s/ns/openshift-marketplace/operators.coreos.com~v1alpha1~CatalogSource/certified-operators`);
+    await browser.wait(until.presenceOf(olmCatalogView.statusBoxTitle('Namespace Not Enabled')));
+    await browser.wait(until.presenceOf(element(by.xpath('//p[contains(text(),"OperatorGroup")]'))));
+    expect(element(by.linkText('Create one here.')).getAttribute('href')).toContain('operators.coreos.com~v1~OperatorGroup/~new');
```

Comment:
I don't see a reason to check the href value. It's better to check if the link is present or click the link and see if it works.